apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -34,6 +39,104 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
       .then(data => setCourseOfferings(data));
   }, []);
 
+  useEffect(() => {
+    // if (!updateSection) return;
+    if (!courseOfferings) return;
+    // const updateCourse = course => updateSection('course', course);
+    if (!isNewSection) {
+      // note that the desire is to come back and refactor this section to
+      // make it more efficient
```

Comment:
I think we could refactor to use a flatten-like function (maybe in a helper file? We might have use for something like that elsewhere). Can be followup work